drivers/SmartThings/zigbee-thermostat/fingerprints.yml

```diff
@@ -13,12 +13,17 @@ zigbeeManufacturer:
     deviceLabel: Danfoss Thermostat
     manufacturer: Danfoss
     model: eTRV0100
-    deviceProfileName: thermostat-popp-danfoss
-  - id: "D5X84YU/eT093WRO"
-    deviceLabel: POPP Thermostat
+    deviceProfileName: thermostat-battery-powerSource
```

Comment:
You have changed the profile used by the danfoss thermostat if I'm reading this diff correctly. Please do not do that.